rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -1608,9 +1608,36 @@ public JobIdData reSubmit(String sessionId, String jobId, PathSegment pathSegmen
 
         WorkflowSubmitter workflowSubmitter = new WorkflowSubmitter(scheduler, space);
         newJobId = workflowSubmitter.submit(tmpJobFile, jobVariables, genericInfos);
+        if (!tmpJobFile.delete()) {
+            logger.warn(tmpJobFile.getAbsolutePath() + " could not be deleted");
+        }
         return mapper.map(newJobId, JobIdData.class);
     }
 
+    @Override
+    public List<JobIdData> reSubmitAll(String sessionId, List<String> jobids) throws RestException {
+        Scheduler scheduler = checkAccess(sessionId, "/scheduler/jobs/resubmit");
+        SchedulerSpaceInterface space = getSpaceInterface(sessionId);
+        WorkflowSubmitter workflowSubmitter = new WorkflowSubmitter(scheduler, space);
+        List<JobIdData> newJobIds = new ArrayList<>(jobids.size());
+
+        for (String jobId : jobids) {
+            try {
+                File tmpJobFile = File.createTempFile("job", "d");
+                String jobContent = scheduler.getJobContent(JobIdImpl.makeJobId(jobId));
+                FileUtils.write(tmpJobFile, jobContent, Charset.forName("UTF-8"));
+                JobId newJobId = workflowSubmitter.submit(tmpJobFile, Collections.emptyMap(), Collections.emptyMap());
+                newJobIds.add(mapper.map(newJobId, JobIdData.class));
+                if (!tmpJobFile.delete()) {
```

Comment:
this condition isn't already checked in submit endpoint ?

Reply:
the delete method returns false when a file cannot be deleted. I don't understand your comment.

Reply:
ok

Reply:
ok